src/charm.py

```diff
@@ -372,6 +372,14 @@ def _prometheus_pebble_layer(self) -> pebble.LayerDict:
                     },
                 }
             },
+            "checks": {
+                "wazuh-alive": {
```

Comment:
```suggestion
                "wazuh-exporter-alive": {
```
Maybe call it `wazuh-exporter-alive` as it's using the exporter port?